### ISSUE TITLE
Unable to remove file with tabulation symbol in its name

### ISSUE THREAD

**Issue by basilefff** [nushell/nushell#12816]
### Describe the bug

When trying to delete files containing tabulation symbol in them, I encountered a very unexpected error.

Also, I did try with freshly compiled version from main, with the same error happening.

### How to reproduce

1. `touch "c\td.txt"`
2. `rm "c\td.txt"`

Alternatively, after first step use auto completion, and even when closed by quotation marks (which by default it isn't) it shows the same error message.

### Expected behavior

Be able to delete file with tabulation symbol in filename

### Screenshots

![screenshot](https://github.com/nushell/nushell/assets/10209879/96743d4e-e167-4fec-9e5c-46a397ddf5c6)

### Configuration

| key                | value                                                            |
| ------------------ | ---------------------------------------------------------------- |
| version            | 0.91.0                                                           |
| branch             | makepkg                                                          |
| commit_hash        | 3016d7a64ccb2c2eac9f735f6144fc896ea724a5                         |
| build_os           | linux-x86_64                                                     |
| build_target       | x86_64-unknown-linux-gnu                                         |
| rust_version       | rustc 1.76.0 (07dca489a 2024-02-04) (Arch Linux rust 1:1.76.0-1) |
| cargo_version      | cargo 1.76.0                                                     |
| build_time         | 2024-03-06 05:55:14 +00:00                                       |
| build_rust_channel | release                                                          |
| allocator          | mimalloc                                                         |
| features           | dataframe, default, extra, sqlite, trash, which, zip             |
| installed_plugins  |                                                                  |

### Additional context

I am pretty sure that the problem is in this code in file nushell/crates/nu-command/src/filesystem/rm.rs at line 160
```Rust
        let corrected_path = Spanned {
            item: match path.item {
                NuGlob::DoNotExpand(s) => {
                    NuGlob::DoNotExpand(nu_utils::strip_ansi_string_unlikely(s))
                }
                NuGlob::Expand(s) => NuGlob::Expand(nu_utils::strip_ansi_string_unlikely(s)),
            },
            span: path.span,
        };
```

Function `nu_utils::strip_ansi_string_unlikely` strips tabs and I am not sure why it is here

**Comment by fdncred** [comment]
I believe double quotes interpret escapes like `\t`. Have you tried with single quotes?

**Comment by sholderbach** [comment]
@basilefff You are correct that the underlying ansi stripping logic removes also most of the ASCII control chars and not explicitly printables (including `\t`)
This will be fixed by #12567 

See the related issue https://github.com/nushell/nushell/issues/6315 and the (flawed) motivation that `find` was inconvenient to use with the fs commands (#6220)

I should finish up my reimplementation of `find` that does highlighting without modifying the actual data and uses metadata instead.

**Comment by basilefff** [comment]
> @basilefff You are correct that the underlying ansi stripping logic removes also most of the ASCII control chars and not explicitly printables (including `\t`) This will be fixed by #12567

Oh, I guess I should have searched the issues more. I guess I shall close this one as a duplicate.

**Comment by sholderbach** [comment]
That's a hard one to find if you are not 100% sure what to search for.

Thanks for reporting it and already drilling down on the underlying problem :)
